build_process/README.md

```diff
@@ -1,27 +1,53 @@
 # Build, Deployment, and Delivery
 
-We believe that the more frequently we deploy, the better we collaborate as
-team and with our clients to navigate our way towards the most appropriate
+We believe that the more frequently we deploy, the better we collaborate as a
+team and with our clients, navigating our way towards the most appropriate
 outcome.
 
-Our expectation is that anyone of our development team should be able to build 
+Our expectation is that anyone on our development team should be able to build 
 and deploy within a few moments of cloning a project repository.
 
+## In This Guide
+* [What is a "build process"?](#what-is-a-build-process)
+* [How we do build processes at Sparkbox](#how-we-do-build-processes-at-sparkbox)
+* [Common build tasks](#common-build-tasks)
+
+### What is a "Build Process"?
+When we use the phrase "build process," we are describing an activity that translates source files into production-ready files. Typically, source code and files are written and created by humans, and the build artifacts—the "executable" code and other assets resulting from a build—are written or compiled via an automated process.
+
+You may hear or see the term "task" or "build task." You can think of a build task as an individual piece of a larger build process. A build task can also serve as a single process that can be executed on its own, outside of the larger build process.
+
+### How We Do Build Processes at Sparkbox
+Our build process is Node-based, meaning that we rely on [Node](https://nodejs.org/en/) to translate our source files into production-ready files. Specifically, our build processes use [npm scripts](https://github.com/sparkbox/standard/blob/master/build_process/node.md#npm-scripts) to handle the translation of source files into build artifacts.
+
+Note that in the past, we have used other Node-based build methods, including [Grunt](https://gruntjs.com/) and [Gulp](https://gulpjs.com/).
+
+Ultimately, regardless of the tools we choose, our goal is to have a build process that fits the projects' needs and our clients' needs, and we strive to strike a balance between having a great developer experience and a great resulting project.
+
+### Common Build Tasks
+Build tasks are used to compile or transpile our source code into production-ready code.
+
+These are some of our common build tasks:
+* Compiling Sass to CSS
+* Adding browser and vendor prefixes to our CSS by using PostCSS with Autoprefixer
+* Transpiling JavaScript
+* Optimizing and/or minifying code
+
 ## Quality Expectations
 
 There are certain things we expect to happen frequently to help keep quality
```

Comment:
I'm not sure I quite understand what this means. Is it something along the lines of, 
"We focus on these things to ensure high quality that is obvious to both our team and our clients."

Reply:
Yeah, I'm not sure this is the world's greatest sentence. This was existing content that I had left in here.

Now that I'm looking at this README with fresh eyes, I think I should take another pass at this, and remove (or radically edit) that Quality Expectations bit, but also include a "Deploy" section and (maybe) a "Delivery" section, as the headline at the very top of the document indicates.

What do you think?

Reply:
From what I see, I think that's a really good idea. I would just suggest running the idea by Adam really quick to make sure.

Reply:
All right, I've updated this to remove the Quality Expectations bit, but also created #205 to note that maybe that stuff should exist elsewhere? I also wrote up pieces on Deployment and Delivery